profile/src/v100/factors/factor_sources/private_hierarchical_deterministic_factor_source.rs

```diff
@@ -0,0 +1,159 @@
+use crate::prelude::*;
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash, uniffi::Record)]
+pub struct PrivateHierarchicalDeterministicFactorSource {
+    pub mnemonic_with_passphrase: MnemonicWithPassphrase,
```

Comment:
FIXME: Remove UniFFI support? It was a rought PoC but I think I'm leaning towards another API, clients can inject entropy to Wallet when init a wallet by creating a new profile. Thus we should never have to care about PrivateHierarchicalDeterministicFactorSource in wallet clients..